airflow/www/extensions/init_views.py

```diff
@@ -220,90 +203,100 @@ def resolve(self, operation):
         return _LazyResolution(self.resolve_function_from_operation_id, operation_id)
 
 
-class _CustomErrorRequestBodyValidator(RequestBodyValidator):
```

Comment:
@RobbeSneyders -  we removed the custom error request body validator completely and rely on validation done by Connexion 3- which i think should be the right call - since Connexion performs the validation now?